third_party/zap/templates/endpoint_config.zapt

```diff
@@ -0,0 +1,4 @@
+{{chip_header}}
+
+
+// TODO issue #3637
```

Comment:
Since this file is unused at the moment maybe we don't even need to include it ?

Reply:
I would keep it as a place holder. Also it tells people that the template is to be completed in a near future.